docs/sources/mimir/operators-guide/deploy-grafana-mimir/getting-started-helm-charts/_index.md

```diff
@@ -93,6 +93,12 @@ Using a custom namespace solves problems later on because you do not have to ove
          # empty, disabled.
    ```
 
+   > **Note:** To see all of the configurable parameters for a Helm chart installation, use `helm show values grafana/mimir-distributed`.
+   > If you want to get the values file from Github, build the URL as follows because we git tag every release:
```

Comment:
I think this is a lot of detail for the getting started guide, I'd move this sentence and the example to the values.yaml